scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/DocumentOps.scala

```diff
@@ -39,7 +40,6 @@ trait DocumentOps { self: DatabaseOps =>
       val isVisited = mutable.Set.empty[g.Tree] // macro expandees can have cycles, keep tracks of visited nodes.
       val todo = mutable.Set[m.Name]() // names to map to global trees
       val mstarts = mutable.Map[Int, m.Name]() // start offset -> tree
-      unit.body.appendMetadata("semanticdbMstarts" -> mstarts) // used in MessagesOps
```

Comment:
Is this intentional?

Reply:
Unrelated change but this metadata was unused

Reply:
How about doing this in a different commit whose message says that it's dead code elimination?